tests/suites/test_suite_ctr_drbg.function

```diff
@@ -2,7 +2,7 @@
 #include "mbedtls/ctr_drbg.h"
 
 int test_offset_idx;
-int mbedtls_entropy_func( void *data, unsigned char *buf, size_t len )
+int mbedtls_test_entropy_func( void *data, unsigned char *buf, size_t len )
```

Comment:
Actually, this function, and `test_offset_idx`, should have been declared `static`.